### PR TITLE
Refactor(web): Improve SCSS coding style and add a comment anytime Stylelint needs to be disabled

### PR DESCRIPTION
**Pull request by adamkudrna**
This is a follow-up of #511.

### CODE REVIEW

File: packages/web/src/scss/components/Header/_Header.scss [hunk 1]
```diff
@@ -244,6 +244,8 @@ button.Header__link--current {
     }
 }
```
Comment (literat) [inline packages/web/src/scss/components/Header/_Header.scss:246]: I prefer adding no new line after block when reenabling stylelinting.

File: packages/web/src/scss/helpers/_animation.scss [hunk 2]
```diff
@@ -1,5 +1,5 @@
-$spin-duration: 1s;
-$spin-easing: linear;
+$-spin-duration: 1s;
```
Comment (literat) [inline packages/web/src/scss/helpers/_animation.scss:1]: What is the difference between using `_` and `-` in variable prefixes? Does it have any meaning?

**Review by literat — CHANGES_REQUESTED** [review]

File: packages/web/src/scss/helpers/_animation.scss (hunk 2 above)
Comment (adamkudrna) [reply packages/web/src/scss/helpers/_animation.scss:1]: They both make the variable private, it's just a syntactic difference coming from the [kebab-case](https://github.com/stylelint-scss/stylelint-config-standard-scss/blob/main/index.js#L51) rule which doesn't allow `_`. (I personally prefer `_` over `-` due to better readability but so far it seems I can live with `-` too.)

File: packages/web/src/scss/components/Header/_Header.scss (hunk 1 above)
Comment (adamkudrna) [reply packages/web/src/scss/components/Header/_Header.scss:246]: You're right, fixed.

File: packages/web/src/scss/helpers/_animation.scss (hunk 2 above)
Comment (literat) [reply packages/web/src/scss/helpers/_animation.scss:1]: Ok, thanks for enlighting me. I also prefer `_` for marking variables private. So I suggest rewriting the regex to allow `_`. What do you think?

Comment (vmilersky) [reply packages/web/src/scss/helpers/_animation.scss:1]: OK and do we actually want to keep `_` allowed only or to keep both chars available? @adamkudrna @literat

Comment (vmilersky) [reply packages/web/src/scss/helpers/_animation.scss:1]: As we agreed, only `_` will be allowed.

Comment (vmilersky) [reply packages/web/src/scss/helpers/_animation.scss:1]: Waiting for merge: https://github.com/lmc-eu/code-quality-tools/pull/106

**Pull request merged**